thoth/storages/graph/postgres.py

```diff
@@ -3565,6 +3571,7 @@ def _create_python_packages_pipfile(
                         os_version=os_version,
                         python_version=python_version,
                         index_url=package.index.url if package.index else None,
+                        metadata=python_package_metadata,
```

Comment:
The called function uses `metadata` just to pick `id`. Can it accept also `metadata_id` and assign `python_package_metadata_id` accordingly? This way we will not issue the query introduced on line 3561:

https://github.com/thoth-station/storages/pull/1387/files#diff-42370be78220d49029e40a3fbfc0f09fR3561

Reply:
Right!! Thanks!!